codegen/table.go

```diff
@@ -37,7 +37,8 @@ type (
 		skipFields           []string
 		structFieldsToUnwrap []string
 
-		unwrapAllEmbeddedStructFields bool
+		unwrapAllEmbeddedStructFields            bool
+		unwrapAllEmbeddedStructFieldsRecursively bool
```

Comment:
```suggestion
		unwrapAllEmbeddedAnonymousStructFieldsRecursively bool
```
just kidding. or am I?